### PR TITLE
add style for right focus pleyer elements

### PR DESCRIPTION
**Pull request by madeira**

### CODE REVIEW

File: video_xblock/static/css/videojs.css [hunk 1]
```diff
@@ -277,10 +277,18 @@ body {
...
 .w-control-bar {
   display: none !important;
 }
+
+.video-js .vjs-control {
+    width: 39px;
+}
+.video-js .vjs-control:focus {
+    outline: rgb(207, 216, 220) solid 1px;
+}
```
Comment (polesye) [inline video_xblock/static/css/videojs.css]: add empty line

File: video_xblock/static/css/brightcove.css [hunk 2]
```diff
@@ -327,3 +328,14 @@ body {
...
+    width: 40px;
+}
+.video-js .vjs-control:focus {
+    outline: rgb(207, 216, 220) solid 1px;
+}
+.vjs-workinghover .vjs-menu-button-popup:focus .vjs-menu,
+.vjs-workinghover .vjs-menu-button-popup:hover .vjs-menu, 
+.vjs-menu-button-popup .vjs-menu.vjs-lock-showing {
+    display: block;
+}
```
Comment (polesye) [inline video_xblock/static/css/brightcove.css]: add empty line

Comment (madeira) [inline video_xblock/static/css/brightcove.css]: empty line was added

File: video_xblock/static/css/videojs.css (hunk 1 above)
Comment (madeira) [inline video_xblock/static/css/videojs.css]: empty line was added

**Pull request merged**